test/index-static-singleton.js

```diff
@@ -0,0 +1,20 @@
+var indexStaticWatcher = require('../lib/index-static-watcher').create();
+
+var indexStatic;
+
+exports.get = function() {
+  if (!indexStatic) {
+    throw new Error("indexStatic has not yet been built.");
+  }
+  return indexStatic;
+};
+
+exports.build = function(done) {
+  this.timeout(10000);
+  indexStaticWatcher.build(function(err, newIndexStatic) {
+    if (err) return done(err);
+
+    indexStatic = newIndexStatic;
+    done();
+  });
+};
```

Comment:
Oh, hm, I think it's also used by `index-static.test.js`? I basically made it so that `index-static.test.js` and `app.test.js` didn't share duplicate code.